osquery/devtools/shell.cpp

```diff
@@ -1086,9 +1090,9 @@ inline void meta_types(struct callback_data* pArg, char* zSql) {
       const auto& name = std::get<0>(column_info);
       const auto& type = columnTypeName(std::get<1>(column_info));
 
-      std::vector<char*> row;
-      row.push_back(const_cast<char*>(name.c_str()));
-      row.push_back(const_cast<char*>(type.c_str()));
+      std::vector<const char*> row;
+      row.push_back(name.c_str());
+      row.push_back(type.c_str());
```

Comment:
Cool! Thanks! Those `const_cast` was killing me :)

nit: you could write it a bit shorter using initialiser list constructor:
```c++
std::vector<const char*> row{{
      name.c_str(),
      type.c_str()
}};
```